.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
         args: [--config, backend/setup.cfg]
       - id: frontend
         name: Yarn Linter
-        entry: yarn --cwd frontend lint
+        entry: bash -c "cd frontend && yarn lint"
```

Comment:
What was the issue with `--cwd`?

Reply:
Behavior was different for some reason in my Windows environment vs a manual cd, --cwd raised a "no penn-clubs/Frontend/Frontend" error, not entirely sure why

Reply:
Oh interesting, well both ways work so no worries with me!

Reply:
Looks like it's a `[known issue](https://github.com/yarnpkg/berry/issues/4908)`?